src/main/kotlin/ornn/res/Strings.kt

```diff
@@ -0,0 +1,14 @@
+object Strings {
```

Comment:
```suggestion
package ornn.res

object Strings {
```
이 파일에 패키지 이름이 생략되어 있습니다!